### DISCUSSION TITLE
Using Measures() for counts

### DISCUSSION THREAD

**Discussion by ciaranmci** [opensafely/documentation#358] [Q&A] [answered]
I think I'm being dumb this morning but the documentation on says they can be used to compute counts. How can Measures() give counts if they are expressed as quotients and a denominator argument needs to be specified?

Can the denominator argument be omitted and defaulted to 1?

**Comment by SRW612** [comment, marked as answer]
You may have solved this by now. But in case you haven't:
I'm not sure if you can omit the denominator argument from the Measure() section of the python code, but the output created by the Measure() function includes the counts (numerator), the population (denominator) and the rate (count/population) as three separate column. So if you just want the counts you can ignore the other columns in the output.

**Reply by ciaranmci** [reply]
That's great. Thanks a mil'.
